pavelib/assets.py

```diff
@@ -41,10 +41,10 @@
 
 # Common lookup paths that are added to the lookup paths for all sass compilations
 COMMON_LOOKUP_PATHS = [
-    path("common/static"),
-    path("common/static/sass"),
+    path('common/static'),
+    path('common/static/sass'),
     path('node_modules'),
-    path('node_modules/edx-pattern-library/node_modules'),
+    path('node_modules/@edx'),
```

Comment:
@arizzitano @efischer19 The challenge is that folks expect to have Sass that looks like this:

    @import 'edx-bootstrap/sass/edx/theme';

I think it is cleaner not to expect the Sass developer to have the extra `@edx/` prefix. I'm opening to changing it if folks would prefer it the other way, but it will require changes to the edx-bootstrap repo.